nixos/modules/services/backup/restic.nix

```diff
@@ -89,6 +89,20 @@ with lib;
             Create the repository if it doesn't exist.
           '';
         };
+
+        pruneOpts = mkOption {
+          type = types.listOf types.str;
+          default = [];
+          description = ''
+            A list of options (--keep-* et al.) for 'restic forget --prune', to automatically prune old snapshots.  The 'forget' command is run *after* the 'backup' command, so keep that in mind when constructing the --keep-* options.
```

Comment:
Can you wrap the lines here for readability in git-like tools?

Reply:
done